sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosItemRequestOptions.java

```diff
@@ -350,6 +361,17 @@ public CosmosItemRequestOptions setDedicatedGatewayRequestOptions(DedicatedGatew
         return this;
     }
 
+    /**
+     * Sets the {@link CosmosEndToEndOperationLatencyPolicyConfig} to be used for the request
```

Comment:
add comment about the default behavior (like picking-up default from client) - same for other request options classes